boto3/dynamodb/types.py

```diff
@@ -190,8 +190,8 @@ def _is_map(self, value):
             return True
         return False
 
-    def _is_list(self, value):
-        if isinstance(value, list):
+    def _is_listlike(self, value):
+        if isinstance(value, list) or isinstance(value, tuple):
```

Comment:
`isinstance` actually takes a tuple with multiple types, so this can be simplified to:
```
if isinstance(value, (list, tuple)):
```